app/src/views/camera/CameraContainer.tsx

```diff
@@ -91,33 +93,66 @@ export default function Camera({ navigation, firestoreCtrl, route }: any) {
             />
 
             <ThemedIconButton
+              onPress={toggleCameraFacing}
+              testID="Switch-Button"
               style={styles.changeOrientationAndFlash}
-              onPress={toggleFlashMode}
-              name={isFlashEnabled ? "flash-off" : "flash"}
+              name="camera-reverse"
               size={24}
               color="white"
-              testID="Flash-Button"
             />
-          </View>
-        </CameraView>
+          </ThemedView>
+        </ThemedView>
       ) : (
-        <View>
-          <ImageBackground
-            source={{ uri: picture?.uri }}
-            style={styles.pictureBackround}
+        <ThemedView style={styles.cameraContainer} colorType="transparent">
+          <ThemedView style={styles.camera} colorType="transparent">
+            <Image source={{ uri: picture?.uri }} style={styles.preview} />
+            <ThemedView style={styles.button} colorType="transparent">
+              <ThemedView style={styles.buttonContainer}>
+                <ThemedIconButton
+                  onPress={toggleLocation}
+                  name={`navigate-circle${isLocationEnabled ? "" : "-outline"}`}
+                  size={35}
+                  color="white"
+                  testID="Location-Button"
+                />
+                <ThemedText colorType="textPrimary">
+                  {isLocationEnabled
```

Comment:
This should be a Boolean!

Reply:
You're generally right but this is already a boolean haha